scss_test.scss

```diff
@@ -0,0 +1,4 @@
+.parent {
+  display: inline-block;
```

Comment:
Line should be indented 4 spaces, but was indented 2 spaces<br>Properties should be ordered background, display